liferay-lima/liferay-portal
# LPS-202233 Change Configuration view for KB

## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPS-202233)

## Changes

Change the configuration for Knowledge Base because it should be open in a new page instead of a modal.

> [!NOTE]
> **You need to activate the LPS-197692 FF.**

## Test Scenario 1

1. Go to Knowledge Base
2. Click on the kebab menu in the top right
3. Navigate to the sections to check the behaviour and styles

![image](https://github.com/liferay-echo/liferay-portal/assets/93203423/c2a6e1e6-1c08-43c9-9221-74b79e2538ec)

cc:// @BarbaraCabrera, @sandrodw3
